src/java/org/sakaiproject/component/app/scheduler/jobs/UCTCourseSiteOwners.java

```diff
@@ -113,15 +114,15 @@ private void addCourseOwners() {
 		int numberOfSites = siteService.countSites(SelectionType.ANY, null, null, null);
 		int first = 0;
 		int last = 0;
-		int pages = numberOfSites/100;
-		int remainder = numberOfSites%100;
+		int pages = numberOfSites/SITE_BATCH_SIZE;
+		int remainder = numberOfSites%SITE_BATCH_SIZE;
 
 		for(int i=0; i<=pages; i++) {
 			if(i == pages ) {
-				first = i*100;
+				first = i*SITE_BATCH_SIZE;
 				last = first+remainder;
 			} else {
-				first = i*100;
+				first = i*SITE_BATCH_SIZE;
 				last = first+99;
```

Comment:
last= first+SITE_BATCH_SIZE-1

Reply:
not the result of the calculation - replace the line 

last = first+99;

with:

last= first+SITE_BATCH_SIZE-1